src/main.ts

```diff
@@ -49,6 +49,9 @@ export type Options = {
   useContext: boolean;
   snakeToCamel: boolean;
   forceLong: boolean;
+  serializers: boolean;
```

Comment:
Overall this looks great; I'm going to bikeshed the names a bit...

Maybe prefix all of them with `output` (or `generate...` but `output...` is shorter) and change to:

* `outputServiceImpl` (I know grpc does use the term "stub" for these, but I think that's a misuse of the term; almost every where else, stub == a test fake)
* `outputEncodeDecodeMethods`
* `outputToFromJsonMethods`

wdyt?

Reply:
@stephenh,
Sounds good, I shortened a bit of your example and came up with this if it's acceptable:
```
export type Options = {
  useContext: boolean;
  snakeToCamel: boolean;
  forceLong: boolean;
  outputEncodeMethods: boolean;
  outputJsonMethods: boolean;
  outputServiceImpl: boolean;
};
```
Just trying to keep them on the shorter side for the command-line.
Close enough?

Reply:
Yep, your names looked good.